go/app/main.go

```diff
@@ -28,14 +39,63 @@ func root(c echo.Context) error {
 func addItem(c echo.Context) error {
 	// Get form data
 	name := c.FormValue("name")
+	category := c.FormValue("category")
 	c.Logger().Infof("Receive item: %s", name)
+	c.Logger().Infof("Receive item: %s", category)
 
 	message := fmt.Sprintf("item received: %s", name)
 	res := Response{Message: message}
 
+	// open json file & data
+	jsonFile, err := os.Open("items.json")
+	if err != nil {
+		log.Fatal("JSONファイルを開けません", err)
```

Comment:
Good job remembering to check errors in Go 👍 

But be careful with `log.Fatal`. It will stop your Go API server 😮 . Instead use `log.Print`.